rpm/rpm.go

```diff
@@ -228,7 +233,7 @@ func buildRPMMeta(info *nfpm.Info) (*rpmpack.RPMMetaData, error) {
 		Licence:     info.License,
 		URL:         info.Homepage,
 		Vendor:      info.Vendor,
-		Packager:    info.Maintainer,
+		Packager:    packager,
```

Comment:
could use
```suggestion
		Packager:    defaultTo(info.RPM.Packager, info.Maintainer),
```

and delete lines 219-223...